src/k8s-extension/azext_k8s_extension/partner_extensions/AzureMLKubernetes.py

```diff
@@ -66,6 +66,11 @@ def __init__(self):
         self.SERVICE_BUS_JOB_STATE_TOPIC = 'jobstate-updatedby-computeprovider'
         self.SERVICE_BUS_JOB_STATE_SUB = 'compute-scheduler-jobstate'
 
+        # constants for enableing SSL in inference
+        self.sslKeyPemFile = 'sslKeyPemFile'
```

Comment:
Consider using a more explicit name, like inferenceEndpointSslKeyPemFile.

Reply:
this name came from Bozhong, I think we can keep it like this